jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -125,11 +126,43 @@ def _parse_as_cube(path):
             header['NAXIS3'] = 1
             header['BUNIT'] = 'dN/s'
             header['CUNIT3'] = 'um'
+            header['CTYPE3'] = 'WAVE'
+
+            # Information not present in the SCI header (was removed by
+            # the pipeline) has to be put back so spectral_cube won't
```

Comment:
The FITS WCS was not removed by the pipeline.  It never existed and was never populated to begin with.  The FITS WCS keywords are not generated for any non-IFU spectral data yet.  But we're actually going to be working on this longstanding issue:

https://github.com/spacetelescope/jwst/issues/2685

next sprint, so there might be FITS keywords very soon in `jwst` master.